napari/util/event.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Vispy Development Team. All Rights Reserved.
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 
-## LICENSE.txt
+# # LICENSE.txt
```

Comment:
smaller diffs, happier colleagues :)

Reply:
agreed, will try and avoid this in the future!